code/modules/mob/mob.dm

```diff
@@ -169,11 +169,11 @@
 // Use for objects performing visible actions
 // message is output to anyone who can see, e.g. "The [src] does something!"
 // blind_message (optional) is what blind people will hear e.g. "You hear something!"
-/atom/proc/visible_message(message, blind_message, max_distance, message_flags = CHAT_TYPE_OTHER)
+/atom/proc/visible_message(message, blind_message, max_distance, message_flags = CHAT_TYPE_OTHER, show_small_icon = FALSE)
```

Comment:
please update the signature of `/mob/visible_message` to match, it's already different enough as it stands